utensor_cgen/backend/operators.py

```diff
@@ -549,10 +571,12 @@ def __init__(self, op_info, **kwargs):
     to_eval = parser.get('to_eval', False)
     self._snippet = QuantRangeForMultiplicationSnippet(inputs, outputs, output_type, ref_counts, to_eval)
 
+
 @OperatorFactory.register
+@OpEqualityDelegate.is_compatible_with("Const", _morphism.Inline2ConstMorphism)
```

Comment:
I like this format

Reply:
Not sure if `morphism` is a good idea.
Not very useful now. Maybe it'll do some cool stuff later (?

Reply:
Assuming the morphisms can chain this could be super useful. Doesnt seem too dangerous

Reply:
Might be useful for stuff like architecture search.
However, we should also define the transform functions morphing from one graph to its equivalences. In addition to `op_type` we should also define how the tensor values need to be modified.

Reply:
Might be useful for stuff like architecture search.
However, we should also define the transform functions morphing from one graph to its equivalences. In addition to op_type we should also define how the tensor values need to be modified.
Seems doable with the current setup.

Reply:
Yes.
But I still have a gut feeling that the current setup may not be good enough.
Need a few iterations of refactoring. We'll see. : )